code/__HELPERS/trait_helpers.dm

```diff
@@ -214,6 +214,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_DEFLECTS_PROJECTILES "trait_deflects_projectiles"
 #define TRAIT_XENO_INTERACTABLE	"can_be_interacted_with_by_xenos"
 #define TRAIT_DODGE_ALL_OBJECTS "dodges_all_objects" /// Allows a mob to dodge all thrown objects
+#define TRAIT_DONT_WRITE_MEMORY "dont_write_memory"
```

Comment:
Is this trait used anywhere? I cant find any other references to it really

Reply:
will remove it was tied to stuff for ians adventure